acceptance/testdata/restorer/container/layers/cacher_buildpack/layer-not-in-cache.toml

```diff
@@ -1,3 +1,3 @@
 build = false
 launch = false
-cache = true
+cache = true
```

Comment:
Do we need a new line at the end of the file?

Reply:
Related convo: https://github.com/buildpacks/lifecycle/pull/582#discussion_r623848168